centreon/src/Core/Broker/Application/Repository/ReadBrokerOutputRepositoryInterface.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Broker\Application\Repository;
+
+use Core\Broker\Domain\Model\BrokerOutput;
+use Core\Broker\Domain\Model\BrokerOutputField;
+use Core\Broker\Domain\Model\Type;
+
+interface ReadBrokerOutputRepositoryInterface
+{
+    /**
+     * Find parameters of an output by type.
+     * Result key is the parameter fieldname.
+     * When $grouped = true,
+     * the groupname is used as a key instead of the fieldname for fields that belong to the same group.
+     *
+     * @param int $typeId
+     * @param bool $grouped
+     *
+     * @throws \Throwable
+     *
+     * @return array<string,BrokerOutputField|array<string,BrokerOutputField>>
+     */
+    public function findParametersByType(int $typeId, bool $grouped = false): array;
```

Comment:
Is the $grouped variable necessary? I don't see any calls with the value false.

Reply:
I thought it might be need it when I made this method, I'll remove it